src/components/Dashboard/TodoList.vue

```diff
@@ -1081,9 +1081,10 @@ export default class TodoList extends Vue {
         // this is only used for internal corrections (not IA):
         correctedFilingId: correction.correctedFilingId,
         // this is only used for external corrections (IA):
-        correctedFilingType: this.filingTypeToName(correction.correctedFilingType as FilingTypes),
+        // Corrections on Alterations always have a correctedFilingType of Alteration -> alterationRequired = true
+        correctedFilingType: this.filingTypeToName(correction.correctedFilingType as FilingTypes, null, true),
```

Comment:
So [`filingTypeToName`](https://github.com/bcgov/business-filings-ui/blob/288cf9465fa3fa6a03e4583b306bc153fe22f1b7/src/mixins/enum-mixin.ts#L229-L230) would set the correctedFilingType as a `CHANGE_OF_COMPANY_INFO` as `alterationRequried` defaults to false. Then [`doResumeFiling`](https://github.com/bcgov/business-filings-ui/blob/288cf9465fa3fa6a03e4583b306bc153fe22f1b7/src/components/Dashboard/TodoList.vue#L1385-L1408) would resume the fling based on the `correctedFilingType` and do a default redirect to a filings UI correction.